pdns/recursordist/settings/table.py

```diff
@@ -40,6 +40,7 @@
 To use this, DNSSEC processing or validation must be enabled by setting :ref:`setting-dnssec` to ``process``, ``log-fail`` or ``validate``.
  ''',
         'versionadded': '4.5.0',
+        'runtime': 'set-max-aggr-nsec-cache-size',
```

Comment:
some of the others have `[]`, is there a difference?

Reply:
a single item gets turned into a list by `generate.py` which takes care of proper formatting and quoting

Reply:
ok :)